tests/hats_import/test_runtime_arguments.py

```diff
@@ -123,15 +123,48 @@ def test_dask_args(tmp_path):
         )
 
 
-def test_provenance_info(tmp_path):
-    """Verify that provenance info ONLY includes general runtime fields."""
+def test_extra_property_dict(test_data_dir):
     args = RuntimeArguments(
-        output_artifact_name="catalog",
-        output_path=tmp_path,
-        tmp_dir=tmp_path,
-        dask_tmp=tmp_path,
-        progress_bar=False,
+        output_artifact_name="small_sky_source_catalog",
+        output_path=test_data_dir,
+    )
+
+    properties = args.extra_property_dict()
+    assert list(properties.keys()) == [
+        "hats_builder",
+        "hats_creation_date",
+        "hats_estsize",
+        "hats_release_date",
+        "hats_version",
+    ]
+
+    # Most values are dynamic, but these are some safe assumptions.
+    assert properties["hats_builder"].startswith("hats")
+    assert properties["hats_creation_date"].startswith("20")
+    assert properties["hats_estsize"] > 1_000
```

Comment:
Nice; I forgot about this ability to improve readability of numeric literals!

Reply:
You're going to get reminded around here. There are a lot of numbers...